src/storage/btree.c

```diff
@@ -11133,7 +11133,7 @@ btree_find_oid_and_its_page (THREAD_ENTRY * thread_p, BTID_INT * btid_int, OID *
   do
     {
       PERF_UTIME_TRACKER_START (thread_p, &ovf_fix_time_track);
-      overflow_page = pgbuf_fix (thread_p, &overflow_vpid, OLD_PAGE, PGBUF_LATCH_WRITE, PGBUF_UNCONDITIONAL_LATCH);
+      overflow_page = pgbuf_fix_read_old_and_check_repl_desync (thread_p, overflow_vpid, PGBUF_UNCONDITIONAL_LATCH);
```

Comment:
This replacement is incorrect as it replaces a write latch with a read one - done internally by `pgbuf_fix_read_old_and_check_repl_desync`.

Reply:
[To be] fixed by https://github.com/CUBRID/cubrid/pull/3299 .